frontend/src/pages/Dashboard.tsx

```diff
@@ -174,14 +170,22 @@ function Dashboard() {
 									</span>
 								</h1>
 								<p style={styles} className={classes.text}>
-									{periodOpen && startDate && endDate ? (
+									{periodStatus === AdmissionPeriodStatus.open &&
+									startDate &&
+									endDate ? (
 										<>
 											<CalendarEvent size={24} strokeWidth={1.5} /> Opptaksperioden er satt
 											fra <span className={classes.date}>{startDate}</span> til{' '}
 											<span className={classes.date}>{endDate}</span>
 										</>
+									) : periodStatus === AdmissionPeriodStatus.finished ? (
+										<span className={classes.date}>
+											Opptaksperioden sluttet {endDate}
+										</span>
```

Comment:
This should maybe have the same formatting as when the admission period is ongoing?

```suggestion
										<>
											Opptaksperioden avsluttet{' '}
											<span className={classes.date}>{endDate}</span>
										</>
```